packages/pay/src/open-payments.ts

```diff
@@ -343,10 +343,13 @@ const validateSpspCredentials = (o: any): PaymentDestination | undefined => {
   }
 }
 
-const validateOpenPaymentsAmount = (o: Record<string, any>): Amount | undefined => {
+const validateOpenPaymentsAmount = (o: Record<string, any>): Amount | undefined | null => {
+  if (typeof o === 'undefined') return
```

Comment:
nit: Pick either `undefined` or `null` to return on error, don't use both.

Also, `o === undefined` would work for this comparison (assuming typescript doesn't complain).

Reply:
Oops, nevermind